### ISSUE TITLE
Default epochs (350) differs from paper (200); train/val/test split not implemented

### ISSUE THREAD

**Issue by talha1369** [szhang1112/scPRS#9]
### Epoch count

The paper Methods state:

> "We trained scPRS for 200 epochs."

But `main.py` (line 86) defaults to 350:

```python
parser.add_argument("--n_epoch", type=int, default=350)
```

### Train/validation/test split

The paper Methods describe:

> "we randomly partitioned the dataset into training, validation and testing sets comprising 60%, 20% and 20% of samples, respectively. We trained different scPRS models with all possible combinations of hyperparameters and assessed their performance (measured by AUROC) on the validation dataset. We selected the model yielding the best performance on the validation set and reported its performance on the held-out test set."

However, `main.py` trains on the **entire dataset** — there is no train/val/test split:

```python
y_train_np = y.detach().cpu().numpy()  # full dataset
# ... upsampling and training on all samples
```

There is no validation-based early stopping or hyperparameter selection in the code.

## Questions

1. Is the 350-epoch default correct for the released code, or should it be 200 as stated in the paper?
2. Was the 60/20/20 split used for the paper results but not included in the released code? If so, could you provide the data splitting and hyperparameter selection code?
3. For the cell prioritization task (100 repeats, training on the full dataset), is the 60/20/20 split bypassed intentionally? The paper seems to describe two separate procedures — hyperparameter selection with splits, then final cell weight estimation on the full data — but this distinction is not clear from the Methods text.
